index.ios.js

```diff
@@ -20,6 +20,9 @@ const store = createStore(
   compose (middleware, autoRehydrate())
 )
 
+// Clear storage when starting the app
+AsyncStorage.clear();
```

Comment:
Please, add a `development` condition to this code. 
This is the common case when you introduce this into develop, you forget at the end of the day and finally it gets deployed, clearing the state from all your users.